src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/verbatim/VerbatimPageObject.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.verbatim;
```

Comment:
Classifying page objects by functionality  not by syntax terms is the way to go. Change the verbatim part of the package name to thirdparties or widgets